draft-enghardt-panrg-path-properties.md

```diff
@@ -73,20 +74,26 @@ Finally, the document lists several path properties that may be useful for the m
 
 Host:
 : A node that generally executes application programs on behalf of user(s), employing network and/or Internet communication services in support of this function, as defined in {{?RFC1122}}.
 
 Link:
 : A medium or communication facility that connects two or more nodes with each other. A link enables a node to send packets to other nodes.
-Links can be physical, e.g., a WiFi network which connects an Access Point to stations, or virtual, e.g., a virtual switch which connects two virtual machines hosted on the same physical machine. A link is unidirectional and bidirectional communication can be modeled as two links between the same nodes in opposite directions.
+Links can be physical, e.g., a Wi-Fi network which connects an Access Point to stations, or virtual, e.g., a virtual switch which connects two virtual machines hosted on the same physical machine. A link is unidirectional. As such, bidirectional communication can be modeled as two links between the same nodes in opposite directions.
 
 Path element:
 : Either a node or a link.
 
 Path:
-: A sequence of adjacent path elements over which a packet can be transmitted, starting and ending with a node. Paths are time-dependent, i.e., the sequence of path elements over which packets are sent from one node to another may change frequently. A path is defined between two nodes. For multicast or broadcast, a packet may be sent by one node and received by multiple nodes. In this case, the packet is sent over multiple paths at once, one path for each combination of sending and receiving node. Note that an entity may have only partial visibility of the path elements that comprise a path, and entities may treat path elements at different levels of abstraction.
+: A sequence of adjacent path elements over which a packet can be transmitted, starting and ending with a node. A path is unidirectional. Paths are time-dependent, i.e., the sequence of path elements over which packets are sent from one node to another may change. A path is defined between two nodes. For multicast or broadcast, a packet may be sent by one node and received by multiple nodes. In this case, the packet is sent over multiple paths at once, one path for each combination of sending and receiving node; these paths do not have to be disjoint. Note that an entity may have only partial visibility of the path elements that comprise a path and visibility may change over time. Different entities may have different visibility of a path and/or treat path elements at different levels of abstraction.
+
+Reverse Path:
+: The path that is used by a remote node in the context of bidirectional communication.
+
+Symmetric Path:
+: Denotes the case where the reverse path can be inferred from the path. For example, if the path is composed of path elements (PE1, PE2, PE3), if the path is symmetric, the reverse path is (PE3, PE2, PE1).
```

Comment:
I like the first sentence but I'm not sure if reverse([PE1,PE2,PE3]) = [PE3,PE2,PE1] is always true since a path element might not have a reverse path element, especially for virtual nodes (e.g., node that drops packets in one direction => no corresponding reverse node).

Maybe we could say that there are different degrees of symmetry, e.g., only a subset of nodes is symmetric (I could imagine an IP symmetric path where nodes describing IP routers that decrease the TTL are symmetric).

Reply:
I would have thought that a path is symmetric if the return path consists of the same path elements, but just in the reverse order/direction, but yes, there might be cases where this is not entirely true. Also, the "same link in the reverse direction" might be a problem sometimes.

Maybe here we could use disjointness, which we define later. So maybe we could move "Symmetric path" to the examples section because it's essentially a special case of disjointness, right?

Reply:
I like this suggestion and added it in 23dd972.
What do you think about the changes?

Reply:
Thanks for moving the definition! I think it makes more sense as a separate item, so I separated it out from the Disjointness definition.
Also, I merged the 'master' branch into this one to resolve the merge conflicts in our Path definition and Access Technology definition.